docroot/sites/all/themes/warmshowers_zen/sass/components/regions/_notification.scss

```diff
@@ -1,11 +0,0 @@
-/***
```

Comment:
@yonas It's not clear to me if we want to remove this file or not. There are a few other SCSS files that use `region-notification`.